.github/workflows/deploy-gh-pages.yml

```
@@ -0,0 +1,69 @@
+# This workflow will publish the documentation to GitHub Pages.
+
+name: ☄️ Deploy Documentation
+
+on: workflow_dispatch
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false

```

will be more cost effective
```suggestion
  cancel-in-progress: true

```

---

This could lead to having incomplete deployments. Isn't it more important to ensure completion, typically in this kind of a simple workflow?